listenbrainz/webserver/views/playlist_api.py

```diff
@@ -328,6 +328,51 @@ def create_playlist():
     return jsonify({'status': 'ok', 'playlist_mbid': playlist.mbid})
 
 
+@playlist_api_bp.route("/create-simple", methods=["POST", "OPTIONS"])
+@crossdomain(headers="Authorization, Content-Type")
+@ratelimit()
+@api_listenstore_needed
+def create_playlist_simple():
+    """
+    Create a quick and dirty public playlist. The POST data must be nothing but a list of recording_mbids
+    separated by , all on one line.
+
+    :reqheader Authorization: Token <user token>
+    :statuscode 200: playlist accepted.
+    :statuscode 400: invalid recording_mbid list sent
+    :statuscode 401: invalid authorization. See error message for details.
+    :statuscode 403: forbidden. The submitting user is not allowed to create playlists for other users.
+    :resheader Content-Type: *application/json*
+    """
+
+    user = validate_auth_header()
+
+    data = str(request.get_data(), "utf-8")
```

Comment:
Since this is POST request, you can instead accept json here. We should do that before we put it in prod. Fine either way for now. 

Maybe also accept the name/description of the playlist optionally here?